NativeImport/ExampleNativeImportFeed.xml

```diff
@@ -79,6 +63,143 @@
+                <NumNegativeFeedbacks>1</NumNegativeFeedbacks>  <!-- Number of negative feedbacks on a given review -->
+                <NumInappropriateFeedbacks>1</NumInappropriateFeedbacks> <!-- Number of times customers reported that this review has inappropriate or irrelevant content -->
+
+                <!-- add a custom question; create submission form question in configuration hub prior to import -->
+
+                <AdditionalFields>
+                      <AdditionalField id="CustomQuestion">
+                        <DisplayLabel>Custom Question:</DisplayLabel>
+                        <Value>This is my custom answer</Value>
+                      </AdditionalField>
+                </AdditionalFields>
+
+
+                <!-- add client responses to reviews -->
+
+                <ClientResponses>
+                  <ClientResponse>
+                    <Department>Responding Department's Name</Department>
+                    <Response>Our number one priority is to deliver exceptional value to our customers every day. We will contact you directly to see how we can improve the outcome for you.</Response>
+                    <ResponseMarkup>
+                      <PlainText>Our number one priority is to deliver exceptional value to our customers every day. We will contact you directly to see how we can improve the outcome for you.</PlainText>
+                    </ResponseMarkup>
+                    <ResponseSource>Workbench</ResponseSource>
+                    <Date>2016-11-12T13:21:43.000-06:00</Date>
+                  </ClientResponse>
+                </ClientResponses>
+
+                <!-- add photos or videos included in the review -->
+
+                <Photos>
+                    <Photo id="104358">
+                        <Caption>Thumbs and some Normal</Caption>
+                        <Sizes>
+                            <Size id="thumbnail" url="https://upload.wikimedia.org/wikipedia/commons/b/b9/Nail(thumb).jpg"/>
+                            <Size id="normal" url="https://upload.wikimedia.org/wikipedia/commons/thumb/5/50/Emoji_Grinning_Face.svg/2000px-Emoji_Grinning_Face.svg.png"/>
+                        </Sizes>
+                    </Photo>
+                </Photos>
+                    
+                <Videos>
+                    <Video>
+                        <Caption>Burgers and Fries</Caption>
+                        <VideoHost>www.youtube.com</VideoHost>
+                        <VideoId>burgersandfries</VideoId>
+                        <RawUrl>https://www.youtube.com/watch?v=DcJFdCmN98s</RawUrl>
+                    </Video>
+                </Videos>
+
+
+                <!-- add context data values to reviews -->
+
+                <ContextDataValues>
+                    <ContextDataValue id="35to44">
+                        <ContextDataDimension id="Age">
+                            <ExternalId>Age</ExternalId>
+                            <Label>Age</Label>
```

Comment:
Is Label required for CDVs and Secondary Ratings? If not, I'd rather leave it out to avoid confusion.